ui/README.adoc

```diff
@@ -0,0 +1,79 @@
+= Hawkular Alerting UI Application
+
+This UI application is built using AngularJS and Patternfly.
+
+link:http://www.patternfly.org/[PatternFly]
+
+link:http://www.patternfly.org/angular-patternfly/#/api[AngularJS + PatternFly]
+
+== Project Layout
+
+The UI source is located under the folder
+
+```
+    hawkular-alerts/ui/src/main/ui
+```
+
+It follows the layout proposed in
+
+link:https://github.com/patternfly/angular-patternfly-demo-app[Angular PatternFly Demo App]
+
+Maven automates the installation of tooling `npm`, `bower` and `grunt`, copy UI resources into `target` folder and prepares
+the distribution that will be hosted under
+
+```
+    hawkular-alerts/ui/target/classes/webroot
+```
+
+and packaged into a `hawkular-alerts-ui-*.jar`.
+
+All content hosted under the `webroot` folder will be served by a Handler under the path
+
+```
+    http://<host>:<port>/hawkular/alerts/ui
+```
+
+== Working on UI
+
+To develop in the UI Application you will need NodeJS tools.
+
+A standard strategy could be to work locally on your
+
+```
+    hawkular-alerts/ui/src/main/ui
+```
+
+directory and prepare the dependencies you need to run all the UI elements
+
+```
+    npm install
+    sudo npm install -g bower
+    bower install
+    sudo npm install -g grunt-cli
+    sudo npm install -g compass
+    gem install sass
```

Comment:
Yes. I don't pretend to be comprehensive here, basically we are pointing to AngularJS + PatternFly and re-using the demo app, so it is possible that some additional note should be added in following iterations.